lib/services/common/securityServiceKeystone.js

```diff
@@ -89,4 +89,26 @@ function getToken(trust, callback) {
     });
 }
 
+/**
+ * Extract the token from the response
+ * @param {String} trust    Trust generated in Keystone to act on behalf of a user.
+ * @param {String} response The authentication response
+ */
+
+function getToken(trust, response, callback) {
+
+    logger.debug(context, 'Retrieving token from the KEYSTONE response');
+
+    if (response.headers['x-subject-token']) {
+        logger.debug(context, 'Token found [%s] for trust [%s]', response.headers['x-subject-token'], trust);
+
+        callback(null, response.headers['x-subject-token']);
+    } else {
+        logger.error(context, 'KEYSTONE-003: Token missing in the response headers', response.headers);
```

Comment:
KEYSTONE-003 is a new error message? Or you are reusing an existing one?

In the case of being a new one, I think we have a place in the documentation where it should be added.

Reply:
new one, i will update doc

Reply:
fixed by 57ede1a